### PR TITLE
MongoDB ssl configs add to set jvm props Fixes #414, fixes for redis ssl

### LINKED ISSUE #414: Add SSL to mongo
It can be set in the URL but add config options for the trust/key store to make it easier to Kubernetes

### PR DESCRIPTION
**Pull request by andrewstevenson**

### CODE REVIEW

File: kafka-connect-redis/src/test/scala/com/datamountaineer/streamreactor/connect/redis/sink/writer/RedisSslTest.scala [hunk 1]
```diff
@@ -59,36 +59,69 @@ class RedisSslTest extends WordSpec with Matchers with BeforeAndAfterAll with Mo
...
-
-  override def beforeAll() = {
-    if (runTests) {
-      setupTrustStore()
-    }
-  }
-
-  override def afterAll() = {
-  }
+//  def setupTrustStore(): Unit = {
```
Comment (spirosoik) [inline kafka-connect-redis/src/test/scala/com/datamountaineer/streamreactor/connect/redis/sink/writer/RedisSslTest.scala]: do we need these? should we remove them?

File: kafka-connect-redis/src/test/scala/com/datamountaineer/streamreactor/connect/redis/sink/writer/RedisSslTest.scala [hunk 2]
```diff
@@ -59,36 +59,69 @@ class RedisSslTest extends WordSpec with Matchers with BeforeAndAfterAll with Mo
...
+      props.get("javax.net.ssl.keyStoreType") shouldBe "JKS"
+
+      props.containsKey("javax.net.ssl.trustStorePassword") shouldBe true
+      props.get("javax.net.ssl.trustStorePassword") shouldBe "truststore-password"
+      props.containsKey("javax.net.ssl.trustStore") shouldBe true
+      props.get("javax.net.ssl.trustStore") shouldBe truststoreFilePath
+      props.containsKey("javax.net.ssl.trustStoreType") shouldBe true
+      props.get("javax.net.ssl.trustStoreType") shouldBe "JKS"
+
+//      if (runTests) {
```
Comment (spirosoik) [inline kafka-connect-redis/src/test/scala/com/datamountaineer/streamreactor/connect/redis/sink/writer/RedisSslTest.scala]: same here

File: kafka-connect-redis/src/test/scala/com/datamountaineer/streamreactor/connect/redis/sink/writer/RedisSslTest.scala (hunk 1 above)
Comment (andrewstevenson) [reply kafka-connect-redis/src/test/scala/com/datamountaineer/streamreactor/connect/redis/sink/writer/RedisSslTest.scala]: its fine

File: kafka-connect-redis/src/test/scala/com/datamountaineer/streamreactor/connect/redis/sink/writer/RedisSslTest.scala (hunk 2 above)
Comment (andrewstevenson) [reply kafka-connect-redis/src/test/scala/com/datamountaineer/streamreactor/connect/redis/sink/writer/RedisSslTest.scala]: yes, I've seen redis use jecks, so we should allow an override, the default is JKS from the configDef

**Pull request merged**